x/staking/keeper/params.go

```diff
@@ -70,9 +70,9 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.ParamsKey)
 	if bz == nil {
-		return params
+		return
```

Comment:
please avoid naked returns 
```suggestion
		return ""
```